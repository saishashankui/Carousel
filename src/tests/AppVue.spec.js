import App from "../App.vue";
import { shallowMount } from "@vue/test-utils";
import axios from "axios";

jest.mock("axios", () => ({
    get: jest.fn(() => Promise.resolve({ data: 3 })),
}));

describe("Mounted App", () => {
    const wrapper = shallowMount(App);

    beforeEach(() => {
        global.$ = jest.fn();
    });

    test("App", () => {
        global.$ = jest.fn();
        expect(wrapper.isVueInstance()).toBeTruthy();
    });

    it("Calls axios.get", () => {
        const result = wrapper.vm.loadImages();
        expect(result).toEqual();
        expect(axios.get).toBeCalledWith(
            "https://www.westelm.com/services/catalog/v4/category/shop/new/all-new/index.json"
        );
    });

    it("loadImages function to be called", () => {
        const output = [
            {
                _id: "hth-gingham-bath-mat-b3589-11",
                active: true,
                img:
                    "https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0362/heather-taylor-home-gingham-bath-mat-1-m.jpg",
                name: "Heather Taylor Home Gingham Bath Mat",
                price: "$40.00",
                thumbnail:
                    "https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0362/heather-taylor-home-gingham-bath-mat-1-m.jpg",
            },
            {
                _id: "hth-scallop-edge-bath-mat-b3587-11",
                img:
                    "https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0260/heather-taylor-home-scallop-edge-bath-mat-m.jpg",
                name: "Heather Taylor Home Scallop Edge Bath Mat",
                price: "$40.00",
                thumbnail:
                    "https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0260/heather-taylor-home-scallop-edge-bath-mat-m.jpg",
            },
            {
                _id: "hth-gingham-shower-curtain-b3590-11",
                img: "#",
                name: "Heather Taylor Home Gingham Shower Curtain",
                price: "$80.00",
                thumbnail: "#",
            },
            {
                _id: "oval-ikat-shower-curtain-b3591-11",
                img: "#",
                name: "Oval Ikat Shower Curtain",
                price: "$90.00",
                thumbnail: "#",
            },
            {
                _id: "organic-stripe-stitch-candlewick-shower-curtain-b3108-11",
                img: "#",
                name: "Organic Stripe Stitch Candlewick Shower Curtain",
                price: "$90.00",
                thumbnail: "#",
            }
        ];
        wrapper.vm.images = [
            {
                flags: [],
                hero: {},
                id: "hth-gingham-bath-mat-b3589-11",
                images: [],
                links: {},
                messages: [],
                name: "Heather Taylor Home Gingham Bath Mat",
                price: {
                    regular: "40",
                    selling: "50",
                },
                reviews: {},
                swatches: [],
                thumbnail: {},
            },
        ];
        const images = wrapper.vm.loadImages();
        expect(wrapper.vm.loadImages()).toEqual(images);
    });

    it("modalshow function to be called", () => {
        wrapper.vm.modalshow("hth-gingham-bath-mat-b3589-11");
        expect(wrapper.vm.isVisible).toEqual(true);
    });

    it("modalhide function to be called", () => {
        wrapper.vm.modalhide();
        expect(wrapper.vm.isVisible).toEqual(false);
    });

    it("renders the correct html", () => {
        expect(wrapper.html()).toMatchSnapshot();
    });
});