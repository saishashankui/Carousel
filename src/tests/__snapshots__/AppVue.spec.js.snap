// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mounted App renders the correct html 1`] = `
<div id="app">
  <div id="data" class="row card-container">
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham Bath Mat</div> <img id="hth-gingham-bath-mat-b3589" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0362/heather-taylor-home-gingham-bath-mat-1-m.jpg">
      <div class="price"> $40.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Scallop Edge Bath Mat</div> <img id="hth-scallop-edge-bath-mat-b3587" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0260/heather-taylor-home-scallop-edge-bath-mat-m.jpg">
      <div class="price"> $40.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham Shower Curtain</div> <img id="hth-gingham-shower-curtain-b3590" src="#">
      <div class="price"> $80.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Oval Ikat Shower Curtain</div> <img id="oval-ikat-shower-curtain-b3591" src="#">
      <div class="price"> $90.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Organic Stripe Stitch Candlewick Shower Curtain</div> <img id="organic-stripe-stitch-candlewick-shower-curtain-b3108" src="#">
      <div class="price"> $90.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham &amp;amp; Stripe Bath Towels</div> <img id="hth-gingham-stripe-bath-towel-b3588" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0258/heather-taylor-home-gingham-stripe-bath-towel-2-m.jpg">
      <div class="price"> $34.50</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Nova Pedal Waste Bin</div> <img id="nova-pedal-waste-bin-b3583" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0200/nova-pedal-waste-bin-1-m.jpg">
      <div class="price"> $110.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Ume Pedal Waste Bin</div> <img id="ume-pedal-waste-bin-b3586" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/ume-pedal-waste-bin-m.jpg">
      <div class="price"> $85.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Nova Toilet Brush</div> <img id="nova-toilet-brush-b3582" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/nova-toilet-brush-2-m.jpg">
      <div class="price"> $69.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Dylan Industrial Bed</div> <img id="dylan-industrial-bed-h8077" src="#">
      <div class="price"> $1,399.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham Bath Mat</div> <img id="hth-gingham-bath-mat-b3589" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0362/heather-taylor-home-gingham-bath-mat-1-m.jpg">
      <div class="price"> $40.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Scallop Edge Bath Mat</div> <img id="hth-scallop-edge-bath-mat-b3587" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0260/heather-taylor-home-scallop-edge-bath-mat-m.jpg">
      <div class="price"> $40.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham Shower Curtain</div> <img id="hth-gingham-shower-curtain-b3590" src="#">
      <div class="price"> $80.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Oval Ikat Shower Curtain</div> <img id="oval-ikat-shower-curtain-b3591" src="#">
      <div class="price"> $90.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Organic Stripe Stitch Candlewick Shower Curtain</div> <img id="organic-stripe-stitch-candlewick-shower-curtain-b3108" src="#">
      <div class="price"> $90.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham &amp;amp; Stripe Bath Towels</div> <img id="hth-gingham-stripe-bath-towel-b3588" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0258/heather-taylor-home-gingham-stripe-bath-towel-2-m.jpg">
      <div class="price"> $34.50</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Nova Pedal Waste Bin</div> <img id="nova-pedal-waste-bin-b3583" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0200/nova-pedal-waste-bin-1-m.jpg">
      <div class="price"> $110.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Ume Pedal Waste Bin</div> <img id="ume-pedal-waste-bin-b3586" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/ume-pedal-waste-bin-m.jpg">
      <div class="price"> $85.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Nova Toilet Brush</div> <img id="nova-toilet-brush-b3582" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/nova-toilet-brush-2-m.jpg">
      <div class="price"> $69.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Dylan Industrial Bed</div> <img id="dylan-industrial-bed-h8077" src="#">
      <div class="price"> $1,399.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham Bath Mat</div> <img id="hth-gingham-bath-mat-b3589" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0362/heather-taylor-home-gingham-bath-mat-1-m.jpg">
      <div class="price"> $40.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Scallop Edge Bath Mat</div> <img id="hth-scallop-edge-bath-mat-b3587" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0260/heather-taylor-home-scallop-edge-bath-mat-m.jpg">
      <div class="price"> $40.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham Shower Curtain</div> <img id="hth-gingham-shower-curtain-b3590" src="#">
      <div class="price"> $80.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Oval Ikat Shower Curtain</div> <img id="oval-ikat-shower-curtain-b3591" src="#">
      <div class="price"> $90.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Organic Stripe Stitch Candlewick Shower Curtain</div> <img id="organic-stripe-stitch-candlewick-shower-curtain-b3108" src="#">
      <div class="price"> $90.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham &amp;amp; Stripe Bath Towels</div> <img id="hth-gingham-stripe-bath-towel-b3588" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0258/heather-taylor-home-gingham-stripe-bath-towel-2-m.jpg">
      <div class="price"> $34.50</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Nova Pedal Waste Bin</div> <img id="nova-pedal-waste-bin-b3583" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0200/nova-pedal-waste-bin-1-m.jpg">
      <div class="price"> $110.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Ume Pedal Waste Bin</div> <img id="ume-pedal-waste-bin-b3586" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/ume-pedal-waste-bin-m.jpg">
      <div class="price"> $85.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Nova Toilet Brush</div> <img id="nova-toilet-brush-b3582" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/nova-toilet-brush-2-m.jpg">
      <div class="price"> $69.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Dylan Industrial Bed</div> <img id="dylan-industrial-bed-h8077" src="#">
      <div class="price"> $1,399.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham Bath Mat</div> <img id="hth-gingham-bath-mat-b3589" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0362/heather-taylor-home-gingham-bath-mat-1-m.jpg">
      <div class="price"> $40.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Scallop Edge Bath Mat</div> <img id="hth-scallop-edge-bath-mat-b3587" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0260/heather-taylor-home-scallop-edge-bath-mat-m.jpg">
      <div class="price"> $40.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham Shower Curtain</div> <img id="hth-gingham-shower-curtain-b3590" src="#">
      <div class="price"> $80.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Oval Ikat Shower Curtain</div> <img id="oval-ikat-shower-curtain-b3591" src="#">
      <div class="price"> $90.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Organic Stripe Stitch Candlewick Shower Curtain</div> <img id="organic-stripe-stitch-candlewick-shower-curtain-b3108" src="#">
      <div class="price"> $90.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Heather Taylor Home Gingham &amp;amp; Stripe Bath Towels</div> <img id="hth-gingham-stripe-bath-towel-b3588" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0258/heather-taylor-home-gingham-stripe-bath-towel-2-m.jpg">
      <div class="price"> $34.50</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Nova Pedal Waste Bin</div> <img id="nova-pedal-waste-bin-b3583" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0200/nova-pedal-waste-bin-1-m.jpg">
      <div class="price"> $110.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Ume Pedal Waste Bin</div> <img id="ume-pedal-waste-bin-b3586" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/ume-pedal-waste-bin-m.jpg">
      <div class="price"> $85.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Nova Toilet Brush</div> <img id="nova-toilet-brush-b3582" src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/nova-toilet-brush-2-m.jpg">
      <div class="price"> $69.00</div>
    </div>
    <div class="card col-sm-12 col-md-4 custom-card">
      <div class="title">Dylan Industrial Bed</div> <img id="dylan-industrial-bed-h8077" src="#">
      <div class="price"> $1,399.00</div>
    </div>
  </div>
  <div id="condition" class="overlay" style="display: none;">
    <div id="text">
      <div class="closeIcon"><span class="closetext">X</span></div>
      <div id="myCarousel" data-ride="carousel" class="carousel slide">
        <ol id="ind" class="carousel-indicators">
          <li data-target="#carouselExampleIndicators" class="active"></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
          <li data-target="#carouselExampleIndicators" class=""></li>
        </ol>
        <div class="carousel-inner">
          <div class="item active"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0362/heather-taylor-home-gingham-bath-mat-1-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0260/heather-taylor-home-scallop-edge-bath-mat-m.jpg" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0258/heather-taylor-home-gingham-stripe-bath-towel-2-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0200/nova-pedal-waste-bin-1-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/ume-pedal-waste-bin-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/nova-toilet-brush-2-m.jpg" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0362/heather-taylor-home-gingham-bath-mat-1-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0260/heather-taylor-home-scallop-edge-bath-mat-m.jpg" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0258/heather-taylor-home-gingham-stripe-bath-towel-2-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0200/nova-pedal-waste-bin-1-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/ume-pedal-waste-bin-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/nova-toilet-brush-2-m.jpg" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0362/heather-taylor-home-gingham-bath-mat-1-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0260/heather-taylor-home-scallop-edge-bath-mat-m.jpg" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0258/heather-taylor-home-gingham-stripe-bath-towel-2-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0200/nova-pedal-waste-bin-1-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/ume-pedal-waste-bin-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/nova-toilet-brush-2-m.jpg" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0362/heather-taylor-home-gingham-bath-mat-1-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0260/heather-taylor-home-scallop-edge-bath-mat-m.jpg" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202116/0258/heather-taylor-home-gingham-stripe-bath-towel-2-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0200/nova-pedal-waste-bin-1-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/ume-pedal-waste-bin-m.jpg" alt="slide"></div>
          <div class="item"><img src="https://assets.weimgs.com/weimgs/ab/images/wcm/products/202117/0282/nova-toilet-brush-2-m.jpg" alt="slide"></div>
          <div class="item"><img src="#" alt="slide"></div>
        </div> <a href="#myCarousel" data-slide="prev" class="left carousel-control"><span class="glyphicon glyphicon-chevron-left"></span> <span class="sr-only">Previous</span></a> <a href="#myCarousel" data-slide="next" class="right carousel-control"><span class="glyphicon glyphicon-chevron-right"></span> <span class="sr-only">Next</span></a>
      </div>
    </div>
  </div>
</div>
`;
